### ISSUE TITLE
Issue building Helk 

### ISSUE THREAD

**Issue by dlohin** [Cyb3rWard0g/HELK#90]
Ran Helk install I kept getting an error after running the build_install.sh

```
[HELK-INSTALLATION-INFO] Setting ES_JAVA_OPTS value...
[HELK-INSTALLATION-INFO] Building & running HELK via docker-compose
 * ERROR: Could not run HELK via docker-compose (Error Code: 1).
```

Upon further inspection I realized it was failing on building Jupyter.

```
Step 13/34 : RUN jupyter labextension install @jupyterlab/hub-extension
 ---> Running in 2fe2fcfa1367
> /usr/bin/npm pack @jupyterlab/hub-extension
jupyterlab-hub-extension-0.9.2.tgz

Errored, use --debug for full output:
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.17.2 <0.18.0        >=0.16.0 <0.17.0 @jupyterlab/apputils
>=2.0.2 <3.0.0          >=1.1.0 <2.0.0   @jupyterlab/coreutils
>=0.17.2 <0.18.0        >=0.16.0 <0.17.0 @jupyterlab/application
The command '/bin/sh -c jupyter labextension install @jupyterlab/hub-extension' returned a non-zero code: 1
```

I was able to fix the issue by specifying a specific version of Jupyterlab.  I went and in history and went back about  a month and it was able to build.  Here is the line I changed in the Dockerfile.

```
# *********** Installing HELK python packages ***************
RUN pip3 install pandas \
  jupyter \
  jupyterlab==0.32.1 \
  jupyterhub
```

**Comment by Cyb3rWard0g** [comment]
Hey @dlohin !! Yes I also had that issue yesterday when I was trying to run some updates. I also ran into the same issue and I was not aware that the latest version of Jupyterlab which was updated yesterday was also in the Jupyter container. that is supposed to have the old one. Anyways I am pushing that change in the next update which will be before BlackHat 👍  Thank you for also sharing what worked for you! I will close this issue in the next commit 👍 Thank you again! I hope you are having a great weekend!
